### ISSUE TITLE
Mark All keyboard shortcut doesn't work unless run as admin

### ISSUE THREAD

**Issue by alexnerd123** [geany/geany#3064]
Windows 10, version 1909
Geany 1.38

When I try to mark all occurrences of a word by highlighting it and pressing ctrl-shift-M (one of my favorite features of geany) it no longer works. Other keyboard shortcuts work fine (ctrl-F and ctrl-H for find and find/replace, for example) so I know geany is receiving my keyboard input (and, you know... its a text editor. It lets my type text lol)

If I highlight a word and right click and use the menus to navigate to Mark All, that works fine. If I right click and use the keyboard to navigate to "search" > "Mark All", that works fine. Its only the keyboard shortcut that fails, and only for this one command (there may be others, but I use this one so often that I noticed very quickly when it stopped working).

Interestingly enough, running Geany as admin fixes the problem. I have no idea if this is a UAC issue??? That doesn't make sense... why would the control and shift keys be "scary"??
I got the idea from troubleshooting another piece of software a while back, greenshot, and it needed to be run as admin to "intercept" the printscreen key in certain cases. That makes enough sense, since printscrn is a global key. But control and shift and M???

Running as admin, unfortunately, isn't "quite" a feasible workaround. Every time I double click a file in explorer to open it in my already-running geany session (as I'm sure you all do all day every day), UAC prompts me if I want to allow geany (from an unverified developer, yadda yadda) to run. I guess thats a "miniature" geany process whos job is detect that theres a main geany running, and shoot the file over to it. I could deal with a UAC prompt once every blue moon, since I honestly leave geany running 24/7, but every time I open a file is a bit much

It does the same thing for both left and right shift and control keys, and I can use them in isolation (ctrl-F works, and shift capitalizes letters, so I know its not a hardware issue)

I even wondered if somehow the combination of ctrl and shift was being treated as a magic, protected, operating system level, key (like printscrn above), but no, ctrl-shift-F for "Find In Files" works just fine. 

I also tried re-binding to a different shortcut, ctrl-shift-N (nancy) and that works fine! Its something about ctrl-shift-M specifically!

So I thought maybe the OS was blocking programs from seeing it. Sure enough, firefox can't see ctrl-shift-M (shortcut for responsive design mode).... but..... chrome can see it just fine (log in as a different user)

I'm honestly quite stumped why this stopped working!! Is there any way to troubleshoot if geany is even receiving the keystrokes?? If this were linux, I would use "xev" or something similar, but this is a windows machine I use for work

**Comment by elextr** [comment]
I don't run Geany on Windows, but one issue that can happen on other systems is that the system grabs some shortcuts for window control before they get to Geany.  Perhaps Windows does that for ctrl+shift+m if you are a user, but not an admin?  Also possibly if your keyboard layout or locale are different from the user to admin.

**Comment by alexnerd123** [comment]
I got lucky on google! Turns out, it was Cisco Webex hogging ctrl-shift-M, for "music mode" (no idea what that is). So Geany and Firefox couldn't see it (unless run as admin). Still, no idea why Chrome was able to see it (best guess is, this is a company machine, and the company might have a special build of Chrome that runs on a higher orbit).

Force-closing webex fixed it. The webex processes kept coming back as I was killing them (so the issue would be fixed for a few seconds, then come back). Some parent PID was respawning them. The powershell-command I ended up running to perma-kill webex was

`taskkill /F /IM ciscowebexstart.exe /T`

And it works now!

Gonna leave this here in case someone else runs into the same problem

https://www.tenforums.com/microsoft-office-365/148737-outlook-hotkey-ctrl-shift-m-not-creating-new-message-window.html
